### PR TITLE
Changelog for multirole so far

### PR DESCRIPTION
**Pull request by timcharper**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-## Changes from 1.8.194 to 1.9.xxx
+## Changes from 1.8.212 to 1.9.xxx
+
+### Multirole support
+
+Marathon 1.9 brings support for multirole, enabling you to launch services for different roles (against different Mesos quotas) with the same Marathon instance.
+
+#### Role field added to services
+
+The role field can now be optionally specified for a service. However, the value of this field may only be sent to 1 of two values:
```
Comment (jeschkies) [inline changelog.md]:
```suggestion
The role field can now be optionally specified for a service. However, the value of this field may only be sent to one of two values:
```

File: changelog.md [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+
+* The default role as specified by `--mesos_role c`ommand line parameter
```
Comment (jeschkies) [inline changelog.md]:
```suggestion
* The default role as specified by `--mesos_role` command line parameter
```

File: changelog.md [hunk 3]
```diff
@@ -1,4 +1,17 @@
-## Changes from 1.8.194 to 1.9.xxx
+## Changes from 1.8.212 to 1.9.xxx
+
+### Multirole support
+
+Marathon 1.9 brings support for multirole, enabling you to launch services for different roles (against different Mesos quotas) with the same Marathon instance.
+
+#### Role field added to services
+
+The role field can now be optionally specified for a service. However, the value of this field may only be sent to one of two values:
```
Comment (ANeumann82) [inline changelog.md]:
```suggestion
The role field can now be optionally specified for a service. However, the value of this field may only be set to one of two values:
```

**Comment by timcharper** [comment]
Closing in favor of #7029

**Pull request closed without merging**